docs/reference/migration/migrate_5_0/scripting.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[breaking_50_scripting]]
+=== Script related changes
+
+==== Indexed scripts and templates
+
+Indexed scrips and templates have been replaced by <<modules-scripting-stored-scripts,stored scripts>>
+which stores the scripts and templates in the cluster state instead of a dedicate `.scripts` index.
+
+Previously indexed scripts in the `.scripts` index will not be used any more as
+Elasticsearch will now try to fetch the scripts from the cluster state. Upon upgrading
+to 5.x the `.scripts` index will remain to exist, so it can be used by a script to migrate
+the stored scripts from the `.scripts` index into the cluster state. The format of the scripts
+hasn't changed.
+
+The following Python script can be used to import your indexed scripts into the cluster state
+as stored scripts:
+
+[source,python]
+-----------------------------------
+from elasticsearch import Elasticsearch,helpers
+
+es = Elasticsearch([
+	{'host': 'localhost'}
+])
+
+for doc in helpers.scan(es, index=".scripts", preserve_order=True):
+	es.put_script(lang=doc['_type'], id=doc['_id'], body=doc['_source'])
+-----------------------------------
+
+This script makes use of the official Elasticsearch Python client and
+therefor you need to make sure that your have installed the client in your
+environment. For more information on this please visit the
+https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/index.html[elasticsearch-py page].
+
+After you have moved the scripts via the provided script or otherwise then you can verify with the following
+request if the migration has happened successfully:
+
+[source,js]
+-----------------------------------
+GET _cluster/state?filter_path=metadata.stored_scripts
+-----------------------------------
+
+The response should include all your scripts from the `.scripts` index.
+After you have verified that all your scripts have been moved, optionally as a last step,
+you can delete the `.scripts` index as Elasticsearch no longer uses it.
```

Comment:
I think it might be a good idea to add a note to the Java API section of the breaking changes doc meantioning the change of method names in the client from IndexScripts to StoredScripts?